hs/src/Reach/Simulator/Core.hs

```diff
@@ -603,6 +603,7 @@ instance Interp DLExpr where
               transferLedger simContract acc ev n
               return V_Null
     DLE_TokenInit _at _dlarg -> return V_Null
+    DLE_CanReceive _at _dlargA _dlargB -> return $ V_Null
```

Comment:
This would return true, because in the simulated network, every account can accept every token

Reply:
(i.e. you just introduced a soundness violation)